persistent/Database/Persist/Sql/Util.hs

```diff
@@ -220,6 +221,19 @@ mkUpdateText' escapeName refColumn x =
   where
     n = escapeName . fieldDB . updateFieldDef $ x
 
+mkPostgresUpdateText :: PersistEntity record => (FieldNameDB -> Text) -> (Text -> Text) -> Update record -> Text
```

Comment:
`postgresql` has different update methods that use the `EXCLUDED` field on the SQL values, so I wrote a method to do operator updates that specific to `postgresql`.  Is this appropriate to use here?

Reply:
If it's specific to postgres, it's probably best for it to live in the `persistent-potgresql` package - `Database.Persist.Postgresql`, perhaps in an `.Internal` module